src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/application/CreateApplicationRestriction.kt

```diff
@@ -0,0 +1,34 @@
+package uk.gov.justice.digital.hmpps.visitscheduler.dto.application
+
+import uk.gov.justice.digital.hmpps.visitscheduler.model.VisitRestriction
+import java.lang.IllegalArgumentException
+
+@Suppress("unused")
+enum class CreateApplicationRestriction(
+  val description: String,
```

Comment:
we could have the VisitRestrition as a parameter here but works as it is currently as well